stablehlo/reference/Element.h

```diff
@@ -38,8 +48,20 @@ class Element {
   /// Returns type of the Element object.
   Type getType() const { return type_; }
 
-  /// Returns the underlying storage of Element object.
-  Attribute getValue() const { return value_; }
+  /// Returns the underlying integer value stored in an Element object with
+  /// integer type.
```

Comment:
What happens if it's we call `getFloatValue` on an integer Element?

Reply:
Made changes so that we will throw a fatal error for those cases.